ihatemoney/forms.py

```diff
@@ -89,26 +89,51 @@ class EditProjectForm(FlaskForm):
     name = StringField(_("Project name"), validators=[DataRequired()])
     password = StringField(_("Private code"), validators=[DataRequired()])
     contact_email = StringField(_("Email"), validators=[DataRequired(), Email()])
+    project_history = BooleanField(_("Enable project history"))
+    ip_recording = BooleanField(_("Use IP tracking for project history"))
 
     def save(self):
         """Create a new project with the information given by this form.
 
         Returns the created instance
         """
+        if not self.project_history.data:
+            new_logging_preference = LoggingMode.DISABLED
+        else:
+            if self.ip_recording.data:
+                new_logging_preference = LoggingMode.RECORD_IP
+            else:
+                new_logging_preference = LoggingMode.ENABLED
+
```

Comment:
This code is duplicated, it could be factorized.

Reply:
You're right, a property method is much cleaner. Fixed in 895ba61d